cifar/yamls/resnet56.yaml

```
@@ -0,0 +1,51 @@
+run_name: cifar10_base # Name of the  training run used for checkpointing and other logging
+is_train: true         # Trains the model if true, otherwise runs evaluation
+seed: 42               # Random seed
+max_duration: 160ep    # Duration to train specified as a Time string
+use_recipe: false      # Whether to train with a recipe for faster convergence or higher accuracy
+grad_accum: 'auto'     # Amount of gradient accumulation, 'auto' (GPU only) means Composer will set the optimal value
+
+train_dataset:
+  path: cifar10
+  is_streaming: false # Set to `true` if your data is in a remote location (e.g. a S3 bucket)
```

```suggestion
  is_streaming: false # Set to `true` if your data is in a remote location (e.g. an S3 bucket)
```
also in the other `is_streaming:` line below